scripts/fetch-mime-support.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+var getMimeTypes = require('./fetch-mime-types-file');
+
+/**
+ * URL for the mime.types file in the Apache HTTPD project source.
+ */
+var URL = 'https://salsa.debian.org/debian/mime-support/-/raw/master/mime.types?inline=false'
```

Comment:
Please make sure to follow up here with the permission of the site operator that we can begin to start polling / scraping their endpoint with an automated process. I couldn't find any public TOS on a quick look, so if there is one that says it's OK, then that's fine and we don't need their explicit permission.

Reply:
Yes agree required for merge. Will do.